packages/plugin-legacy/src/index.ts

```diff
@@ -545,7 +545,7 @@ function viteLegacyPlugin(options: Options = {}): Plugin[] {
         configFile: false,
         compact: !!config.build.minify,
         sourceMaps,
-        inputSourceMap: undefined, // sourceMaps ? chunk.map : undefined, `.map` TODO: moved to OutputChunk?
```

Comment:
We don't need to pass input sourcemap because rollup will handle sourcemap merging.